scripts/pipelines/azure-devops/templates/common/check-pipeline-exec.sh

```diff
@@ -0,0 +1,46 @@
+#!/bin/bash
+################################ Description ############################################
+# This script is used to get the runId and the result of specific execution of a pipeline
+# using the name of the pipeline and the commit on which the pipeline should be executed
+################################# Arguments #############################################
+# $1 = Name of the pipeline
+# $2 = Commit on which the execution you are looking for has been played
+################################# Output ################################################
+# runId : The Id of the run, you can use it in your pipeline into any task (for example to dowload an artifact from a build pipeline using this Id)
+# result : The result of the execution you found, it can be 'canceled', 'failed' or 'succeeded'
+################################# Additional infos ######################################
+# There is a var called "number_lst", it limits the size of the list of execution,
+# feel free to change this value depending on your project
+#########################################################################################
+
+# Init var
+Pipeline_to_find="$1"
+sourceVersion="$2"
+i=0
+number_lst=5000

```

Comment:
```suggestion
pipelineToFind="$1"
sourceVersion="$2"
i=0
pipelineRunsListLimit=100
```

You should be consistent with naming convention used. Mixing is not good, as with spirit drinks :)

Rename references.